tests/Tests/400_ods_dashboard/400_ods_dashboard.robot

```diff
@@ -33,6 +33,24 @@ Verify Content In RHODS Explore Section
     Check Number Of Displayed Cards Is Correct  expected_data=${EXP_DATA_DICT}
     Check Cards Details Are Correct   expected_data=${EXP_DATA_DICT}
 
+Verify Documentation Link Https status code
+    [Tags]  Sanity
+    ...     ODS-327  ODS-492
+    Click Link    Resources
+    Sleep  2
+    #get the documentation link
+    ${href_view_the_doc}=  Get Element Attribute    xpath=//*[@id="root"]/div/div[2]/div/div/div[1]/div/main/div[1]/div/div/div/section/div/p/a    href
+    ${status_for_view_the_doc}=  Get HTTP Status Code    ${href_view_the_doc}
```

Comment:
hm where are taking the keyword `Get HTTP Status Code` from? Are you sure it shouldn't be `Check HTTP Status Code` like at like 50?